azure-pipelines/templates/test.yml

```diff
@@ -1,4 +1,26 @@
 steps:
+  # This gives the AzCodeE2ETests service connection access to this pipeline.
+  - task: AzureKeyVault@1
+    displayName: 'Authorize AzCodeE2ETests service connection'
```

Comment:
Can you prepend the `displayName` with this emoji: 👉
It's little but it helps visually distinguish which steps in the pipeline are ours vs. coming from the 1ES pipeline templates.

```suggestion
    displayName: '\U0001F449 Authorize AzCodeE2ETests service connection'
```

Reply:
e.g. 
![image](https://github.com/microsoft/vscode-azuretools/assets/36966225/2ae6f8fa-776f-43fc-bc47-1d2c1f9766c5)

Reply:
Done!